clearwater-snmpd/usr/share/clearwater/infrastructure/scripts/snmpd_fd_handler

```diff
@@ -33,10 +33,10 @@ if [ "$1" = "-n" ]; then
     $oid_prefix | $oid_prefix.0 | $oid_prefix.0.* | $oid_prefix.1)
       oid_req=$oid_prefix.1.0;;
 
-    $oid_prefix.1.* | $oid_prefix.2 | $oid_prefix.2.0)
+    $oid_prefix.1.* | $oid_prefix.2)
```

Comment:
I spotted that this code incorrectly returned .2.0 when doing a get next on .2.0 (similarly for .3.0)

Reply:
Good spot. I should have thought of that